### ISSUE TITLE
9.4 Phase 0 Reflection

### ISSUE THREAD

**Issue by kelly-keating** [phase-0/phase-0#3537]
# 9.4 Phase 0 Reflection

Reference the [reflection resource](https://github.com/dev-academy-programme/curriculum/tree/master/resources/nt-reflection-ARTICLE) to get the most out of this assignment.

- [x] Start Toggl.
- [x] Using the command line, navigate to your cloned 'phase-0' repository and find the `ee` folder.
- [x] Pull down any changes.
- [x] In your `ee/yourname-lastname` directory create a file called `9-reflection`
- [x] Open in VS Code and record your answers:
  - [x] What did I do well **through my nine sprints in Phase 0**?
  - [x] What could I have done to improve **through my nine sprints in Phase 0**?
  - [x] What learnings and habits have I developed that will help me in Bootcamp?
- [x] Stage and commit with a meaningful message. Push to GitHub.
- [x] Post a link to your reflection file on GitHub in the waffle card comments below.

**Comment by leslie-alldridge** [comment]
What did I do well through my nine sprints in Phase 0?

I knew starting out, that the biggest challenge for me was going to be finding that balance between work, life, study and other hobbies. The first few weeks it was evident that I'd not yet found the right balance - I was focusing too much on study or feeling over worked and avoiding study. 

As I developed and learned more throughout the sprints, my self management improved and I managed to identify my priorities and bring balance into my life. I'm really proud that I've managed to get to sprint 9 earlier than intended and I think my previous coding experience has helped a lot. I'm overjoyed that I bought kindness, effort and integrity to every sprint and submitted work that was above what was asked of me, on time. 

Lastly, somehow...the last nine sprints have really improved my life at work and at home. I'm feeling better now, than I was back at the start of 2018.
